inc/search.class.php

```diff
@@ -4613,15 +4790,17 @@ static function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val, $meta
     * @param string $ref_table            Reference table
     * @param array &$already_link_tables  Array of tables already joined
     *
-    * @return string Left join string
+    * @return string Left join SQL string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addDefaultJoin($itemtype, $ref_table, array &$already_link_tables) {
+   public function addDefaultJoin($itemtype, $ref_table, array &$already_link_tables) {
```

Comment:
This method is used in some plugins (`formcreator` and `fusioninventory` for instance), change should be listed in changelog.